### DISCUSSION TITLE
求助  :  openresty安装报错     stderr: service "openresty" has no container to start

### DISCUSSION THREAD

**Discussion by XCQ0607** [1Panel-dev/1Panel#4847] [Q&A]
1Panel版本:v1.10.6-lts
浏览器:Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36 QuarkPC/1.3.5.36

尝试过重装了，但是docker界面就是没有openresty的容器
错误:stderr: service “openresty” has no container to start

截图：
![90b6aa73bea7a7b5fa52da34ce8698fb8ef0cb49_2_690x335](https://github.com/1Panel-dev/1Panel/assets/125746521/b1469df0-3a67-4f6d-a1a2-d2a3a6605a21)
![8d50266f6a89625fbe2ab5009e6a87eff150b8f1_2_690x335](https://github.com/1Panel-dev/1Panel/assets/125746521/969521f7-ccbf-494f-a7ff-8dff5e50b2fe)

**本人半路小白，感谢耐心回答我的问题！！

如果需要更多的信息，在评论区要求我提供就可以，谢谢!

论坛问题地址：https://bbs.fit2cloud.com/t/topic/5065

**Comment by wanghe-fit2cloud** [comment]
进入 `容器` - `镜像` 页面，查看一下应用镜像是否已经下载成功，如图：
![image](https://github.com/1Panel-dev/1Panel/assets/46734380/05901083-8632-4776-8607-c7e1969da85b)

> 如果未能搜索到 openresty 镜像，可以考虑进入 `容器` - `配置` 页面，添加一下镜像加速，然后再进入 `应用商店` - `已安装` 页面，`重建` 应用。
